timemachine/lib/potentials.py

```diff
@@ -427,6 +427,20 @@ def get_cutoff(self):
         return self.args[3]
 
 
+class NonbondedPairListPrecomputed(NonbondedCustomOpWrapper):
```

Comment:
Sure. Note that this is placeholder still - the C++ equivalent for this is not done. I was going add this in a later pass. There's some nuance here I should probably document - we can't actually use this class for the "cancellation of NaNs", since the floating point operations will be different from NonbondedAllPairs